pkg/resources/cloudcontroller/digitalocean.go

```diff
@@ -0,0 +1,129 @@
+package cloudcontroller
+
+import (
+	"fmt"
+
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/apis/kubermatic/v1"
+	"k8c.io/kubermatic/v2/pkg/resources"
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+	"k8c.io/kubermatic/v2/pkg/resources/vpnsidecar"
+	appsv1 "k8s.io/api/apps/v1"
+	_ "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
+	_ "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	DigitalOceanCCMDeploymentName = "digital-ocean-cloud-controller-manager"
```

Comment:
Nit: It's usually `digitalocean`, not `digital-ocean`.

```suggestion
	DigitalOceanCCMDeploymentName = "digitalocean-cloud-controller-manager"
```